README.md

```diff
@@ -1,10 +1,57 @@
 
 ## Project prerequisites
 Make sure that you have the compatible versions of Visual Studio with .NET MAUI workloads and .NET SDK version in your machine before starting to work on this project.
 
 ## How to run this application
 To run this application, you need to first clone the `getting-started-with-.net-maui-accordion` repository and then open it in Visual Studio 2022. Now, simply build and run your project to view the output.
+
+## Features and Benefits
+
+### Data binding
+The Accordion control allows the use of .NET MAUI [BindableLayout](https://help.syncfusion.com/maui/accordion/bindablelayout?utm_source=github&utm_medium=listing&utm_campaign=maui-accordion-github-samples) to bind a data collection and populate the control.
+
+### Expand and collapse
+The Accordion control allows users to interactively and programmatically customize the expand and collapse actions.
+
+### Animation
+Enhances the user experience with animated transitions for item expansion and collapse. The animations can be customized by turning them on or off, adjusting the duration of the expanding and collapsing animations, and modifying the easing style.
+
+### Visual state manager
+Modify the appearance of Accordion items according to their expanded and collapsed states by utilizing the [visual state manager](https://help.syncfusion.com/maui/accordion/appearance#visual-state-manager?utm_source=github&utm_medium=listing&utm_campaign=maui-accordion-github-samples).
+
+### Expander icon position
+The Accordion provides the flexibility to customize the placement of the header icon in accordion items.
+
+## Related links
+[Learn More about .NET MAUI Accordion](https://www.syncfusion.com/maui-controls/maui-accordion?utm_source=github&utm_medium=listing&utm_campaign=maui-accordion-github-samples)
+
+[Download Free Trial](https://www.syncfusion.com/downloads/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-accordion-github-samples)
+
+[Pricing](https://www.syncfusion.com/sales/teamlicense?utm_source=github&utm_medium=listing&utm_campaign=maui-accordion-github-samples)
+
+[Documentation](https://help.syncfusion.com/maui/accordion/getting-started?utm_source=github&utm_medium=listing&utm_campaign=maui-accordion-github-samples)
+
+[View Demos](https://github.com/SyncfusionExamples/getting-started-with-.net-maui-accordion?utm_source=github&utm_medium=listing&utm_campaign=maui-accordion-github-samples)
```

Comment:
need to change view demos and online example link

Reply:
resolved